src/types.ts

```diff
@@ -33,11 +33,14 @@ export interface Driver<OptionsT = any, InstanceT = any> {
     commonOptions?: TransactionOptions
   ) => MaybePromise<{ key: string; value: StorageValue }[]>;
   /** @experimental */
-  getItemRaw?: (key: string, opts: TransactionOptions) => MaybePromise<unknown>;
+  getItemRaw?: (
+    key: string,
+    opts?: TransactionOptions
```

Comment:
[core sets it](https://github.com/unjs/unstorage/blob/main/src/storage.ts#L164) when calling drivers.

If in `Storage` it is not optional i think that is the mistake